Libraries/LibJS/Runtime/Uint8ClampedArray.h

```diff
@@ -39,15 +39,14 @@ class Uint8ClampedArray final : public Object {
 
     i32 length() const { return m_length; }
 
-    virtual bool put_by_index(u32 property_index, Value value) override;
+    virtual bool put_by_index(u32 property_index, Value value, bool) override;
```

Comment:
It's a bit unclear what the "bool" part is here (even though it comes from the override).
```suggestion
    virtual bool put_by_index(u32 property_index, Value value, bool throw_exceptions) override;
```

Reply:
Well my thinking was that because it was ignored in the implementation of the function, it should be "ignored" in the declaration of the function too. But I can still add it if you think it makes more sense